lib/src/container/mod.rs

```diff
@@ -267,6 +267,23 @@ pub struct ManifestDiff<'a> {
     pub added: Vec<&'a oci_spec::image::Descriptor>,
 }
 
+/// Struct to export the manifest diff
+#[derive(Debug)]
+pub struct ExportedManifestDiff {
+    /// Total number of packages
+    pub total: String,
+    /// Size of total number of packages.
+    pub total_size: String,
+    /// Number of packages removed
+    pub n_removed: String,
+    /// Size of the number of packages removed
+    pub removed_size: String,
```

Comment:
Shouldn't these be integer values e.g. `u64`?

Reply:
I've changed them all the `u64` and made the appropriate changes on the rpm-ostree side.